app/controllers/application_controller.rb

```diff
@@ -14,4 +16,17 @@ def authenticate_admin!
       redirect_to request.referrer || root_path, notice: "Admins Only! :|"
     end
   end
+
+  def send_slack_notification msg
+    uri = URI('https://slack.com/api/chat.postMessage')
+    params = { :token => Rails.application.secrets.slack_token, :channel => Rails.application.secrets.slack_channel, :text => msg, :pretty => 1 }
+    uri.query = URI.encode_www_form(params)
+
+    res = Net::HTTP.get_response(uri)
+    if res.is_a?(Net::HTTPSuccess)
+      puts "Slack response: #{res.body}"
```

Comment:
There is no logger in application.rb so does all puts go to heroku logging?

Reply:
Thanks for the feedback. Log removed.